js/renovation/ui/scroll_view/bottomPocket.tsx

```diff
@@ -0,0 +1,44 @@
+import {
+  JSXComponent,
+  OneWay,
+  ComponentBindings,
+  Component,
+  Ref,
+  RefObject,
+} from 'devextreme-generator/component_declaration/common';
+
+import {
+  SCROLLVIEW_BOTTOM_POCKET_CLASS,
+} from './scrollable_utils';
+
+const SCROLLVIEW_REACHBOTTOM_CLASS = 'dx-scrollview-scrollbottom';
+const SCROLLVIEW_REACHBOTTOM_INDICATOR_CLASS = 'dx-scrollview-scrollbottom-indicator';
+const SCROLLVIEW_REACHBOTTOM_TEXT_CLASS = 'dx-scrollview-scrollbottom-text';
+
+export const viewFunction = (viewModel: BottomPocket): JSX.Element => {
+  const {
+    bottomPocketRef,
+  } = viewModel;
+
+  return (
+    <div className={SCROLLVIEW_BOTTOM_POCKET_CLASS} ref={bottomPocketRef}>
+      <div className={SCROLLVIEW_REACHBOTTOM_CLASS}>
+        <div className={SCROLLVIEW_REACHBOTTOM_INDICATOR_CLASS} />
+        <div className={SCROLLVIEW_REACHBOTTOM_TEXT_CLASS}>Updating...</div>
```

Comment:
"Updating..." - let's renovate all texts later in a separate PR, simply don't delete the text assigning code from toDelete

Reply:
Fix in 3527be22a9377543d60df6a6071aec8f72421780